src/machines/modelingMachine.ts

```diff
@@ -1460,7 +1522,7 @@ export const modelingMachine = setup({
Lryb4aNkMSSKlY6qvhXQtAeDUx+wigeZixmQUhmrqIBxagmzkyWyKE5L6bkVUUmB3K2FqaYSJAoaSr6UmAgq+BuMgRHRgReow4yhiVkZ+Yd1BjWRGodrpV1b3K1ElbTqVa6aJr1agrbqkbz8LR9YlhRAYQ7RTw+VUgcViyLQbAhlxaqT7lpb0qwA5UFVcQlVJYIaxrHaLrcAq0Y72Mc9tbVilBkLByUw6ILwLQ0ghxrIeogQ7siagFI6U71obVsIsA7bCrk7oamA667U7rvgrBDVmIOQcg+a5JDki59FbA6gmQFyKCmT-BcpSDMBiKaq2bRNrcSwHAeaUZEMchFg4J9hpEDU2Jmz7kiAJ7ezp6yCFj57e9Khx4LS+7yIVrSxJAmDt60bfgZ9w74VD7KCT7Z6qBrBWbe98w9gFAfhjhJN16H6t7DlshEwky36Y4P7J6v6pCZA-7-wclcxJxRBwRuo2QwHN74Id7fNX790D6j7ZZEGqAVAUGBdf1IRXIzQ4RnQUxEN8gEY1guYS53JK6KF4Hj6Z6pCxAqGHMaGHL7BzQ3i4JaK7JbAFh7xzRYpX9ewFzsASQ2LmSezmTulJyNBpzVBqIWoPMhBm1jQ0jwQPV6hfqJS14iBlHVHvyPTfzHZvT-zALyAbHAzHHgy3bbIhlojvqTZLymGHNIKW0YL9FiwzQlGVGuyZ73TcMnYXwSAbZ+gdztTbg3GDTWTyjUZpBe1XVdG5yDHjTZBcwnKlB1D1YGcuGJBrGonZTMztScyanVG57MmjSchcwrIiEmroRLpvHRdvg4JWQdhhR4cw7iGHF-BcAz0AZ-RfBAoAgMB5otowjxoMmfcjTBAy5xwUDahARfgYQClmRUYdhuR9nzE0LXCyBsAuhhgnYMV2cJpG6tKrmbmRggqZnNGUhP1wUZwWro8ClRFDxOr77vZpN96Xnbnp17nVnXdW93cTMIW3nipBAPmvG0EN6LzkZf18hTxDxjFqyrBorXNilFHLmJDXm7mz0CpqsnZ6xaYnmGNEXK1kW6XZ00XEArRmRTREoopxJhzJHGD54fZERfN5gxmzUvICB8RZpZoggVmfpulapej7BEZfbJJCm7cjRlh2ojlDVtYXBmgpmMB4AogqTz7-xBBQ99Y4IagVi3j6hQQRBMhjQZsTm4RHAYHiGvJzWaoglrWck7XTthLlYORvh7ofhRd87no-RBphpvoJoTX1nSKRAMp-XbX0og3EN+xH8mQM0lhGo1Bo2aM3pPp424AfXYYhY8xeElrKKNAroMWK8Rbrl5DyEK3ZhahHWX7HJWYQ4QdgHxV5Flx23vEMhjdkoMpgR4RzGLwrzjpmI+1EJKdPWAN59bZgaqtf9lsR2KiN0REGG3UMwRMdjySDUuYRULGUzYmQtrqz5j8SNOMd35IFgMhNi2YDjWQxtvjLJfZDVZtAsqm5LHtt3WnSLSyHQTQCSRFpxO3IchnSJOZJiBaeRGdkdmcbMn3Cl9ZhRGrJKRV+7thMhAW-GUKrzHd0zH3QPWVWZHR2rdGdXg3f4HQbAS4zR6g+NG847U9KOk3qOFJagRapwwJydIcRc8He6cxPgxTYHZKAbQCv82lwsH2MOqO3YsHIQnKahDhorRloN9kgHNWXD974V3C7in3RVN0HBb8WoWpzRQRUw0g7CHKC3UgahtCci3S5in3M1LPJwSlbOCPGidjQVexy9Mt5yqnTOYmpnjMc4d3hZCF2otrxJ5h4LbImjNA4p7zzFolIuPOYm8iCi8r4uJlpBbX2Y7RctbCLA8xBl8g7REDL3Jb9COAd2QJFJ9lUjFDAveYnNHMu7hZchjjTjOAz5LjT0d39HTIglNAKVOR7OSIoNsTX8e6mv7kkSQShgwTzYISn38TrWJl7cRsDnbIxRPZ-UBOuqI5jPnTCL0yzPVPesIJN6ChTxNiK9CmbwEZaguZzoLRpPxnbvPzom9CfyoA1RlTgjNz1T4vS4XuUpAQp57JQREJTI69qhUvoRC2qnOy1z7Hwe-zYf+TxA6hMhskORAumi4zlJEzZFcfsLdyczcfYeq4A46OERBl-Z6Jkpu6dgiEsD0Klz6eHveP4xBAtjvGp9jQBNPqck1v36MLheYmwfVSYfHuiIaB4qx4yVkLixUL5e4HFe7uvy3SVfCf1eDpNfvGS4zE+NynCtKTAfXyjfge6mcLIxGmXfVyd2reSh5BSx0gRtUieC2PWL2LkUuKn3GrrWA52ZCwLBENJdRMfgHB7wEzeqT1lKQq-RMON7J5OQtAxGFCbKtWhRkYNBRBcCbvZOSb0qyrsq41E388jTb8jQ6GzA7AXNXqRmBmagl4w4APq-ibbYZbFMprBqShRf85HoxEVXmp50VrTg2+WP4QgRBzruJaI6AbR-GbMOA-2-5+vraJ6ICFkxYRO-LbXEN2KaEahIe9p-9-cg3ifZCwTkahHIVXGozQORxWZLh+Agd+t7VZJTTv5n4uEM-OhotTzTIxJGjgHYNa2UAnRnCRDVdjXxH7pVraGKNhLn0OwD4Q4TlAbsSjqDG4GygzXRiuz-5V1t+GAuWjbRwEadl0OwSSELD5RzVC0WCKxClAN5oCABUddOoki46Rg9+bfJ-lANFI9c2QuYBRr+hnAQpseQ-KgbXxrqt0lQ9dTALnx0Sc1-MpwZYD13Lyex9gVoOwloBQGUDuGpDKejPXi5rEGC4IWPBlC4HMNwG+DZ+nvU37v10mIhHduL2DbwR7qmDeuHrSyhVMmmIPUQmbw8agDYCiAX3tsDZC+NzyevMjiEM8Gm98euZRJtgH6A+9jEKQI5qpE5DFgKmjvVAVY1SExN6muFfwKELAA5DbI5geEIHwXaAgQ+dQchJM2mYLRahFvWYL5gqBWUZy94aSPiyHJOUjg-YDQGWDJbXNIW4PaFj9Da6iIpI4IH2q5ykgFJTIgzDMGLScjZREWlLUGHwBpaKlzYO7GcGcn4SgpeCllezl83vAsRVghiA1k4CAA */
```

Comment:
Why are some of the sketch tools in this random spot down the bottom?
![image](https://github.com/user-attachments/assets/28150659-c728-4f60-add6-fd2fcddbd5a6)

Reply:
Oh this probably was someone who doesn't us the vscode editor caused this to happen prior to you right?

Reply:
Oh let me make it pretty real quick